meetings/2023-02-15.md

```diff
@@ -45,6 +51,28 @@
+* ML: Sébastien Rosset has been working on ActivityPods (with ActivityPub) and HTTP Signature in the context of notifications and suggested using proxy and http signature here: https://forum.solidproject.org/t/authentication-with-proxy-and-http-signature/5517. 
+* SC: Good point. https://github.com/solid/notifications/issues/148
+* SC: Had similar thoughts on what we can do to get a server to be able to send a message to another server, where the flow starts with client. Client talks to server, and then server talks to another server. That was the case in the LDN channel type, but it overlaps with the case mentioned by ML, and overlaps with work Henry Story on HTTPSig has been doing. That's just one part when signing these messages. Other part is exchanging the keys. If we can align these patterns and considerations it seems like we're looking at the same direction. 
+* eP: We can discuss details now. HTTP Signatures is a valid direction. Some nuances: we currently work with Solid-OIDC with the assumption that the resource server shouldn't be too complex, and should have its own AuthN server. We leave it to implementation how to handle access tokens. We just specify the ID Token and ways to interact with AuthZ server to push id token in exchange for access token. The complexity can be on AuthZ server and resource server can focus on other things. With HTTP Signatures the Resource Server may need to  ... my main intention is to find a way to move forward and addressing those nuances with these use cases. Jackson proposed something, there are other approaches. In that work we should not separate AuthZ and AuthN. Maybe task force or dedicated meetings, they should be worked on together because they are hand in hand. 
```

Comment:
```suggestion
* SC: Had similar thoughts on what we can do to get a server to be able to send a message to another server, where the flow starts with client. Client talks to server, and then server talks to another server. That was the case in the LDN channel type, but it overlaps with the case mentioned by ML, and overlaps with work Henry Story on HTTPSig has been doing. That's just one part when signing these messages. Other part is exchanging the keys. If we can align these patterns and considerations, it seems like we're looking at the same direction. 
* eP: We can discuss details now. HTTP Signatures is a valid direction. Some nuances: we currently work with Solid-OIDC with the assumption that the resource server shouldn't be too complex, and should have its own AuthN server. We leave it to implementations how to handle access tokens. We just specify the ID Token and ways to interact with AuthZ server to push id token in exchange for access token. The complexity can be on AuthZ server, and resource server can focus on other things. With HTTP Signatures, the Resource Server may need to  ... my main intention is to find a way to move forward and address those nuances with these use cases. Jackson proposed something; there are other approaches. In that work, we should not separate AuthZ and AuthN. Maybe task force or dedicated meetings; they should be worked on together, because they are hand in hand. 
```